src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -132,16 +132,6 @@
         <target state="translated">Našlo se několik polí typu Microsoft.Extensions.Logging.ILogger</target>
         <note>{Locked="Microsoft.Extensions.Logging.ILogger"}</note>
       </trans-unit>
-      <trans-unit id="PrimaryConstructorParameterLoggerHiddenMessage">
```

Comment:
This is undoing a change from May.
https://github.com/dotnet/runtime/commit/9daa4b41eb9f157e79eaf05e2f7451c9c8f6dbdc#diff-2b526fe79ad2825b0cb16698c883ce0f55f5c11973e0a4a745c4c987d1a9b515R135